src/engine/ruleset/integrations/windows/rules/execution-python-script-in-cmdline.yml

```diff
@@ -0,0 +1,38 @@
+name: rule/execution-python-script-in-cmdline/0
+
+metadata:
```

Comment:
Isn't it necessary to add the compatibility section?

Reply:
ditto